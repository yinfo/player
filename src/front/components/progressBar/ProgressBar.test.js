// @flow

import React from 'react';
import { shallow } from 'enzyme';
import ProgressBar from './ProgressBar';

describe('ProgressBar component', () => {
  it('renders as expected', () => {
    const props = {
      barSize: 'progress',
      barStriped: false,
      barStyle: 'primary',
      active: false,
      valueNow: 0,
      valueMin: 0,
      valueMax: 100,
      screenReadersText: 'progress',
    };

    const component = shallow(<ProgressBar {...props} />);
    expect(component).toMatchSnapshot();
  });
});
