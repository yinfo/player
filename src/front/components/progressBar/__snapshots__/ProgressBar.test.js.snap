// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProgressBar component renders as expected 1`] = `ShallowWrapper {}`;
